TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Management/AssetController.php

```diff
@@ -255,4 +258,25 @@ protected function getErrorFlashMessage()
         }
         return new Error($errorMessage, null, [get_class($this), $this->actionMethodName]);
     }
+
+    /**
+     * Add a translated flashMessage.
+     *
+     * @param string $messageBody The translation id for the message body.
+     * @param string $messageTitle The translation id for the message title.
+     * @param string $severity
+     * @param array $messageArguments
+     * @param integer $messageCode
+     * @return void
+     */
+    public function addFlashMessage($messageBody, $messageTitle = '', $severity = Message::SEVERITY_OK, array $messageArguments = array(), $messageCode = null)
+    {
+        $localeObject = $this->_localizationService->getConfiguration()->getCurrentLocale();
```

Comment:
this is redundant when using "BackendUserTranslationTrait", that was the point.. $localeObject can just be null in the `translateById` calls